lib/mods/replace-methods.js

```diff
@@ -30,7 +30,8 @@ function isNamespaceMethod(node, baseTypeName) {
    *
    * First declared as CSS, with a baseTypeName of CSS (Namespace ✔)
    */
-  if (firstDeclaration && firstDeclaration.getInitializer) {
+  if (firstDeclaration && firstDeclaration.getInitializer
+        && firstDeclaration.getInitializer()) {
```

Comment:
Why was this change introduced in this PR?

Reply:
Will move to refactor PR